SimCalorimetry/HGCalSimProducers/src/HGCFEElectronics.cc

```diff
@@ -314,6 +353,22 @@ void HGCFEElectronics<DFr>::runShaperWithToT(DFr &dataFrame, HGCSimHitData& char
   };
   runChargeSharing();
 
+
+  /*
```

Comment:
delete commented-out code or add comment why it should be kept

Reply:
This part I would keep and postpone, since the digitizer will need major restyling in the coming months, and then it should be more clear how to deal with toa for out of time signals. But if it is a problem I can delete.